app/com/arpnetworking/metrics/portal/query/impl/KairosDbQueryExecutor.java

```diff
@@ -106,7 +106,7 @@ public KairosDbQueryExecutor(final KairosDbService service, final ObjectMapper o
         try {
             metricsQueryBuilder = _objectMapper.readValue(query.getQuery(),
                     com.arpnetworking.kairos.client.models.MetricsQuery.Builder.class);
-        } catch (final JsonProcessingException e) {
```

Comment:
Similarly here, according to [these docs](https://fasterxml.github.io/jackson-databind/javadoc/2.9/com/fasterxml/jackson/databind/package-tree.html) `JsonProcessingException` is in `2.9`

Reply:
You are correct; however, `readValue` in 2.9 throws `IOException` and in 2.10 doesnot.

https://fasterxml.github.io/jackson-databind/javadoc/2.9/com/fasterxml/jackson/databind/ObjectMapper.html#readValue-java.lang.String-com.fasterxml.jackson.databind.JavaType-

vs

https://fasterxml.github.io/jackson-databind/javadoc/2.10/com/fasterxml/jackson/databind/ObjectMapper.html#readValue-java.lang.String-com.fasterxml.jackson.databind.JavaType-